src/domain_bridge/generic_subscription.hpp

```diff
@@ -69,6 +69,11 @@ class GenericSubscription : public rclcpp::SubscriptionBase
   void handle_loaned_message(
     void * loaned_message, const rclcpp::MessageInfo & message_info) override;
 
+  void
+  handle_serialized_message(
+    const std::shared_ptr<rclcpp::SerializedMessage> & serialized_message,
+    const rclcpp::MessageInfo & message_info) override;
```

Comment:
If we remove the `override` does that make it backwards compatible with Galactic?

Reply:
Yep!  That totally works.  I'm going to put a comment here as well explaining why we aren't marking it as override.